content/pt-pt/docs/setup/learning-environment/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Ambiente de aprendizagem
+weight: 20
+---
+
+<!--
+{{/* Existe um redirecionamento do Netlify desta página para /docs/tasks/tools/ */}}
+{{/* O conteúdo desta página existe apenas para fornecer um stub de navegação */}}
+{{/* e para proteger no caso de esse redirecionamento ser um dia removido. */}}
+
+{{/* Se está a localizar esta página, só precisa de copiar o front matter */}}
+{{/* e adicionar um redirecionamento em "/static/_redirects", para a SUA localização. */}}
```

Comment:
Make sure to add that redirect. And you can drop everything after line 4 once you've done this.